oap-server/server-starter/src/main/resources/ui-initialized-templates/pulsar/pulsar-root.json

```diff
@@ -0,0 +1,37 @@
+[
```

Comment:
This root dashboard doesn't have a widget pointing to the doc of latest. Please refer to the other existing dashboards.

Reply:
Fixed.